lib/python/rosie/ws.py

```diff
@@ -109,7 +109,7 @@ def index(self, *_):
         """Provide the index page."""
         try:
             return self._render()
-        except:
+        except (KeyError, AttributeError, jinja2.exceptions.TemplateError):
```

Comment:
`jinja2.exceptions.TemplateError` might be sufficient here?

Reply:
Most likely. The client will get a 500 in the worst case.